### PR TITLE
Academics page

### PR DESCRIPTION
**Pull request by nigow**
### Ticket
[Ticket Link](Link)

### What is changed / added
- short description of the PR

### Screenshots
- if applicable

### CODE REVIEW

File: src/api/academic/content-types/academic/schema.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    },
+    "creator": {
+      "type": "string",
+      "required": true
+    },
+    "cover_image": {
+      "type": "media",
+      "multiple": false,
+      "required": true,
+      "allowedTypes": [
```
Comment (TheFatPanda97) [inline src/api/academic/content-types/academic/schema.json:27]: let's only allow images here
Screen Shot 2022-03-08 at 8 22 21 PM

File: src/api/academic/content-types/academic/schema.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      ]
+    },
+    "content": {
+      "type": "richtext",
+      "required": true
+    },
+    "featured": {
+      "type": "boolean",
+      "required": false
+    },
```
Comment (TheFatPanda97) [inline src/api/academic/content-types/academic/schema.json:40]: let's make this required and default to false
Screen Shot 2022-03-08 at 8 23 15 PM

**Review by TheFatPanda97 — CHANGES_REQUESTED** [review]
noice, just some small changes (keep in mind you can edit the field settings in the GUI, no need to edit the code 😃)

File: src/api/academic/content-types/academic/schema.json (hunk 2 above)
Comment (nigow) [reply src/api/academic/content-types/academic/schema.json]: good catch, changed both fields

**Review by TheFatPanda97 — APPROVED** [review]

**Pull request merged**